src/lib/swapper/swappers/ThorchainSwapper/utils/addAggregatorAndDestinationToMemo.ts

```diff
@@ -0,0 +1,34 @@
+import type { Address } from 'viem'
+
+import { MEMO_PART_DELIMITER } from './constants'
+
+export const addAggregatorAndDestinationToMemo = ({
+  quotedMemo,
+  aggregator,
+  destinationToken,
+  minAmountOut,
+}: {
+  quotedMemo: string | undefined
+  aggregator: Address
+  destinationToken: Address
```

Comment:
Non-blocking naming improvements here:

- We don't conventionally use the `token` terminology (except for the specific case of checking is an asset is a token vs. a native asset)
- Using the address terminology, we make it clear that this is an asset address, though types definitely help already

```suggestion
  destinationAssetAddress: Address
```

Alternatively, `receiveAssetAddress` would match our swapper terminology, or `finalAssetAddress` would be another option if we want to stick to [THOR docs](https://dev.thorchain.org/aggregators/memos.html) vernacular

Reply:
I gone for `finalAssetAddress` as sticking to the thor doc doesn't seems like a bad idea, I would be happy with both our swapper term or thor one, as you prefer

Reply:
`finalAssetAddress` is absolutely fine, tyvm!